model/dao/TemplateDAO/PostgreSQLTemplateDAO.php

```diff
@@ -188,7 +192,7 @@ public function delete(TemplateVO $templateVO) {
         }
 
         // Otherwise delete a task.
-        if(sizeof($currTaskVO) > 0) {
+        if($currTaskVO->getId()) {
```

Comment:
This should be a null check, in case you try to delete a template that doesn't exist, line 191 will return null and line 195 will crash.

Reply:
Good catch, I just fixed this